### ISSUE TITLE
Service discovery - support adminState 

### ISSUE THREAD

**Issue by jrooty** [F5Networks/f5-appsvcs-extension#759]
This feature request is related to the service-discovery engine. 

When deploying a service using the service discovery, it seems that adminState parameter is not implemented when "addressDiscovery" : "consul".

I don't know if this limitation is extended to other service discovery mode. 

 
The need is related to multiple K8S clusters, where a Pool has to reach multiple service discovery services/multiple clusters.

A Pool reaches the cluster1 (service-discovery1). The retrieved members are enabled ("adminState": "enable).

The same Pool reaches the cluster2 (service-discovery2). the retrieved members are disabled/offline ("adminState": "offline|disable").

Currently, changing the "adminState" has no effect.

This feature will be very usefull in a multi K8S Cluster environement (HA at K8S cluster level).

 
I understand that action will take effect on all service-discovery members.

 
Example :

```
"members": [

                    {

                        "adminState": "enable",

                        "servicePort": 443,

                        "addressDiscovery": "consul",

                        "autoPopulate": true,

                        "uri": https://myservice-discovery1.f5.com/,

                        "updateInterval": 300,

                        "rejectUnauthorized": false,

                        "shareNodes": true

                    },

                     {

                        "adminState": "offline",

                        "servicePort": 443,

                        "addressDiscovery": "consul",

                        "autoPopulate": true,

                        "uri": https://myservice-discovery2.f5.com/,

                        "updateInterval": 300,

                        "rejectUnauthorized": false,

                        "shareNodes": true

                    }

                ]

```

**Comment by jrooty** [comment]
Hello,
To be efficient, it could be interesting to know when "fqdn" is set for addressDiscovery, if that feature (adminState) is supported or not.
If not, we could include this usecase inside this ER.
Thank you.

**Comment by sunitharonan** [comment]
Closing as this has been released in AS3 3.48.0
